lib/cloud_controller/rack_app_builder.rb

```diff
@@ -21,6 +21,7 @@ def build(config, request_metrics)
         use CloudFoundry::Middleware::Cors, config.get(:allowed_cors_domains)
         use CloudFoundry::Middleware::VcapRequestId
         use CloudFoundry::Middleware::NewRelicCustomAttributes if config.get(:newrelic_enabled)
+        use Honeycomb::Rack::Middleware, client: Honeycomb.client if config.get(:honeycomb, :write_key)
```

Comment:
i feel like this ought to also be conditional on `config.get(:honeycomb)`

Reply:
Yes. Updated.